src/actions/nodePoolActions.js

```diff
@@ -154,10 +196,10 @@ export function nodePoolsCreate(clusterId, nodePools) {
       nodePools.map(nodePool => {
         return nodePoolsApi
           .addNodePool(clusterId, nodePool)
-          .then(nodePool => {
+          .then(pool => {
```

Comment:
Wondering here as well about the variable renaming. We don't really ever talk about `pools` or `pairs`

Reply:
Same here